GitUI/UserControls/RevisionGrid/Graph/RevisionGraphRow.cs

```diff
@@ -2,37 +2,31 @@
 
 namespace GitUI.UserControls.RevisionGrid.Graph
 {
-    public interface IRevisionGraphRow
```

Comment:
[nit] personally I prefer an interface to live in the same file as the implementing type, unless there is more than one implementation. I find it makes it easier to modify the API.

Reply:
moved back